webapp/components/abstract-test-file-results-table.js

```diff
@@ -6,70 +6,73 @@
 import { TestRunsBase } from './test-runs.js';
 import { WPTColors } from './wpt-colors.js';
 import './test-file-results.js';
-const $_documentContainer = document.createElement('template');
+import { html } from '../node_modules/@polymer/polymer/lib/utils/html-tag.js';
```

Comment:
```suggestion
import { html } from '@polymer/polymer/polymer-element.js';
```
Same problem as last time.

Reply:
So, this link is copied from another file. It's strange to me that it's broken in some files and that goes unnoticed - I'll send a separate PR for changing all of them.

Reply:
Yeah that's indeed strange. Looks like it's only broken when you directly import `html-tag.js` in *tests*. Might have something to do with the dynamic import path overwrite of `wct-server`.

Reply:
https://github.com/web-platform-tests/wpt.fyi/pull/1157